src/components/filterSection/FilterInput.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function SearchBox(props){
+
+    const { item } = props;
+    const { className, img, inputClass, inputType, label, inputPlaceholder } = item;
+
+    return(
+        <div className={`inline filter-inputs ${className}`}>
+            {img && <img src={img} alt={label} />}
+            {inputClass && <input className={inputClass} type={inputType} placeholder={inputPlaceholder}/>} 
```

Comment:
This line doesn't make sense to me because if someone don't want to pass className from outside so they'll not get the input field. Think about it

Reply:
++